build.gradle

```diff
@@ -1,33 +1,44 @@
+plugins {
+    id 'java'
+    id 'org.gretty' version '3.0.1'
+}
+
 group 'com.twilio'
 version '1.0'
 
-apply plugin: 'java'
-apply plugin: 'war'
-apply from: 'https://raw.github.com/akhikhl/gretty/master/pluginScripts/gretty.plugin'
-
 gretty {
     httpPort = 8080
     contextPath = ''
     logbackConfigFile = "src/main/resources/logback.xml"
     jettyXmlFile = "src/main/resources/jetty-logback-access.xml"
 }
 
-sourceCompatibility = 1.8
+sourceCompatibility = 11
 
 repositories {
-    mavenCentral()
+    jcenter()
 }
 
 dependencies {
-    compile 'ch.qos.logback:logback-classic:1.2.2'
-    compile 'ch.qos.logback:logback-access:1.2.2'
-    compile 'javax.servlet:javax.servlet-api:3.1.0'
-    compile 'org.eclipse.jetty:jetty-annotations:9.3.6.v20151106'
-    compile 'com.twilio.sdk:twilio:7.0.0-rc-29'
+    def logbackVersion='1.2.3'
+    implementation(group: 'javax.servlet', name:'javax.servlet-api', version: '4.0.1')
+    implementation(group: 'javax.servlet', name:'jstl', version:'1.2')
+    // implementation(group: 'org.eclipse.jetty', name:'jetty-annotations', version: '9.4.29.v20200521')
```

Comment:
If this one is not longer needed, we can delete it :slightly_smiling_face:

Reply:
Good catch! removed.